hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFileStatus.java

```diff
@@ -190,4 +196,87 @@ public void testGetFileStatusDoesNotResumeRenameOnNonAtomicDir() throws Exceptio
         .getRedoRenameAtomicity(Mockito.any(Path.class), Mockito.anyInt(),
             Mockito.any(TracingContext.class), Mockito.nullable(AbfsLease.class));
   }
+
+  @Test
```

Comment:
small comments on what each test does would be nice

Reply:
Added